tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
   "compilerOptions": {
     "baseUrl": ".",
-
+    "paths": {
+      "ui/*": ["src/ui/public/*"]
```

Comment:
I think that will actually break serverside TS code, that needs to import `ui`?

But I am not very sure about how that is supposed to work, so maybe @elastic/kibana-platform could have a look at that? It feels for me, that we might want to add `src/ui/*` maybe too?

Reply:
I thought, that there are a couple of folders outside of `src/ui/public` inside `src/ui` that might need to be required from serverside code outside of `src/ui` itself. Within `src/ui` it should use relative paths, but if you are needing to import anything from that folder from a (core_)plugin we would require `ui/` to work I think. But yeah it was just a vague idea, that's why I wanted someone from platform to look into it, so clarify that I was mistaken :)

Reply:
Got it, it seems server code in core plugins relies only these ugly `../../../(phew)/../../ui/*` relative imports currently :) And x-pack plugins don't depend on server part of `src/ui` directly.

Reply:
Yeah, we don't do any require remapping server side, so the only thing that will do a plain `import 'ui/...` is public code.